src/index.js

```diff
@@ -38,10 +39,10 @@ function createFormAction (requestAction, types, payloadCreator = identity) {
     throw new Error('Must include two action types: [ SUCCESS, FAILURE ]');
   }
 
-  return Object.assign((data, dispatch) => {
+  return Object.assign((data, dispatch, meta) => {
```

Comment:
@TeaBough thanks for the PR! This function typically gets passed as a callback to `handleSubmit`, which then [calls](https://github.com/erikras/redux-form/blob/master/src/handleSubmit.js#L39) it with `values`, `dispatch` and `props`

Reply:
@mhssmnn what about `Object.assign((data, dispatch, props, meta) => { ...` ?

Reply:
> @TeaBough thanks for the PR! This function typically gets passed as a callback to `handleSubmit`, which then [calls](https://github.com/erikras/redux-form/blob/master/src/handleSubmit.js#L39) it with **`values`, `dispatch` and `props`**

@mhssmnn The README only states `values` and `dispatch`, so I guess `props` should be added as well. I had a similar issue regarding this one.